src/doc/rustdoc/src/command-line-arguments.md

```diff
@@ -417,6 +417,12 @@ When `rustdoc` receives this flag, it will print an extra "Version (version)" in
 the crate root's docs. You can use this flag to differentiate between different versions of your
 library's documentation.
 
+## `-`: load source code from the standard input
+
+If you specify `-` on the command line, then `rustdoc` will read until EOF from the
+stdin (standard input) for the source code, instead of reading from the filesystem
+with an otherwise specified PATH.
```

Comment:
```suggestion
If you specify `-` as the INPUT on the command line, then `rustdoc` will read the
source code from stdin (standard input stream) until the EOF, instead of the file
system with an otherwise specified path.
```

Dunno, like this maybe? rustdoc calls the “path argument” `INPUT` not `PATH` (cf. `rustdoc -h`). `PATH` could be confused with the env var `PATH`.

Reply:
Well, technically speaking reading from STDIN is reading from the file system, too, on POSIX systems (STDIN is a file to, cf. `/dev/stdin`).

Reply:
> Dunno, like this maybe? rustdoc calls the “path argument” INPUT not PATH (cf. rustdoc -h). PATH could be confused with the env var PATH.

Sure, sound goods. Updated

> Well, technically speaking reading from STDIN is reading from the file system, too, on POSIX systems (STDIN is a file to, cf. /dev/stdin).

Regular file? :smile: